src/tests/ftest/io/single_server_mpiio.yaml

```diff
@@ -0,0 +1,62 @@
+hosts:
+   test_machines:
+        test_servers:
+            - boro-A
+        test_clients:
+            - boro-B
+            - boro-C
+            - boro-D
+            - boro-E
+            - boro-F
+            - boro-G
+            - boro-H
+            - boro-I
```

Comment:
This is why it failed in CI:

```
17:35:02  Error: Placeholders missing replacements in ./io/single_server_mpiio.yaml:
17:35:02    boro-I
```

My fault I guess for suggesting you add it to the `Test-tag:` pragma.